core/controllers/profile_test.py

```diff
@@ -282,3 +284,32 @@ def test_profile_data_is_independent_of_currently_logged_in_user(self):
         response = self.get_json(
             '/profilehandler/data/%s' % self.EDITOR_USERNAME)
         self.assertEqual(response['user_bio'], 'My new editor bio')
+
+
+class FirstContributionDateTests(test_utils.GenericTestBase):
+
+    USERNAME = 'abc123'
+    EMAIL = 'abc123@gmail.com'
+
+    def test_contribution_datetime(self):
+        """Test the contribution date shows up correctly as nonexist."""
+        self.signup(self.EMAIL, self.USERNAME)
+        self.login(self.EMAIL)
+        user_services.get_or_create_user(self.USERNAME, self.EMAIL)
+        self.user_id = self.get_user_id_from_email(self.EMAIL)
+        response = self.testapp.get('/profile/%s' % self.USERNAME)
```

Comment:
Don't need this line and the next one. The two requests are independent and in this test you're really only testing profilehandler/data.